Areas/Identity/Data/ApplicationUser.cs

```diff
@@ -12,6 +12,10 @@ public class ApplicationUser : IdentityUser
     public string FirstName { get; set; } 
     
     public string LastName { get; set; }
+
+    public string Email { get; set; }
```

Comment:
We don't need that email and password here. ApplicationUser class extends Identity user class which already provide us with fields such us: password, login, username etc. In ApplicationUser class we specify only these fields that are not provided by identity user.